packages/big-design-icons/package.json

```diff
@@ -42,35 +42,35 @@
     "styled-components": "^5.3.5"
   },
   "devDependencies": {
-    "@babel/cli": "^7.15.4",
-    "@babel/core": "^7.15.5",
-    "@babel/plugin-transform-runtime": "^7.15.0",
-    "@babel/preset-env": "^7.15.6",
+    "@babel/cli": "^7.24.1",
+    "@babel/core": "^7.24.4",
+    "@babel/plugin-transform-runtime": "^7.24.3",
+    "@babel/preset-env": "^7.24.4",
     "@babel/preset-react": "^7.12.1",
     "@babel/preset-typescript": "^7.15.0",
-    "@bigcommerce/configs": "^0.20.1",
+    "@bigcommerce/configs": "workspace:^",
     "@styled/typescript-styled-plugin": "^1.0.1",
     "@svgr/core": "^8.1.0",
     "@svgr/plugin-jsx": "^8.1.0",
     "@svgr/plugin-prettier": "^8.1.0",
     "@svgr/plugin-svgo": "^8.1.0",
     "@types/react": "^18.2.73",
     "@types/react-dom": "^18.2.23",
```

Comment:
There's an issue with the latest `@types/react` + `@types/react-dom` that doesn't mesh well with `@types/styled-components`. For some reason `0n` is being passed into as a falsey value, which may be related with [this lodash type](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/80f7e6bdd99074bf98ce4c00a25d2ffc949d71f7/types/lodash/common/array.d.ts#L37). I'm not entirely sure, but I avoided it for now.